spec/mailers/admin_mailer_spec.rb

```diff
@@ -18,11 +18,12 @@
   describe '#member_application_received' do
 
     let(:new_app) { create(:membership_application, user: test_user)  }
-    let(:email_sent) { AdminMailer.member_application_received(new_app) }
+    let(:admin) { create(:user, email: 'admin@example.com', admin: true) }
+    let(:email_sent) { AdminMailer.member_application_received(new_app, admin) }
 
     it_behaves_like 'a successfully created email',
                     I18n.t('application_mailer.admin.new_application_received.subject'),
-                    ENV['SHF_MEMBERSHIP_EMAIL'],
+                    'admin@example.com',
                     '' do
       let(:email_created) { email_sent }
     end
```

Comment:
You need to add specification for ` it 'has a previewer' ...` (use the spec in the `membership_application_spec.rb` as an example)

You need to add specifications for the other important parts of the email so that future maintainers understand that the information is required in the email, and so that it can serve as tests to ensure that this and future code is correct.  Ex:
- it has the correct  member_app.user.full_name
- it has the correct member_app.contact_email
- it has the correct url_for(@member_app)
- it has the text for the new app has arrived (the t('....'))
- it has the text for the view app here (the t('....'))
- it has the text for must be logged in (the t('....'))